packages/backend/src/videos/videos.dto.ts

```diff
@@ -28,5 +29,21 @@ export class FilterVideoDTO implements IFilterVideoRequest{
     title: string;
     
     @ApiProperty()
-    category: string; 
+    category: string;
+
+    @ApiProperty({required:false})
```

Comment:
Justyna tez to zrobiła. Dawaj znac który bierzesz task i przeucaj z backlogu do sprintu

Reply:
był nie przypisany